tests/plugins/os/unix/log/test_messages.py

```diff
@@ -105,3 +106,21 @@ def test_unix_log_messages_malformed_log_year_rollover(target_unix_users, fs_uni
 
         results = list(target_unix_users.messages())
         assert len(results) == 2
+
+
+def test_unix_messages_cloud_init(target_unix_users: Target, fs_unix: VirtualFilesystem) -> None:
+    messages = """
+    2005-08-09 11:55:21,000 - foo.py[DEBUG]: This is a cloud-init message!
+    2005-08-09 11:55:21,001 - util.py[DEBUG]: Cloud-init v. 1.2.3-4ubuntu5 running 'init-local' at Tue, 9 Aug 2005 11:55:21 +0000. Up 13.37 seconds.
+    """
+    fs_unix.map_file_fh("/var/log/installer/cloud-init.log", BytesIO(textwrap.dedent(messages).encode()))
+
+    target_unix_users.add_plugin(MessagesPlugin)
+
+    results = list(target_unix_users.messages())
+    assert len(results) == 2
+    assert results[0].ts == dt.fromtimestamp(1_1_2_3_5_8_13_21)
```

Comment:
I like what you did here ;). But it seems that the test fails.

```suggestion
    assert results[0].ts == dt(2005, 8, 9, 11, 55, 21)
```